cvrf2csaf/cvrf2csaf.py

```diff
@@ -106,8 +134,10 @@ def convert_file(self, path) -> dict:
 
         self._parse(root)
 
-        return self._compose_final_csaf()
+        if SectionHandler.error_occurred:
+            critical_exit("Some error occurred during parsing the document, can't produce output.")
```

Comment:
Given the time I review this, I might have missed something. But shouldn't that depend on the `--force` config option..

Reply:
Yes, thanks. Will be fixed.